### PR TITLE
Add content in post-installation procedure

### LINKED ISSUE #2117: Post Cluster deployments documentation
**Component**:

**Why this is needed**:

Once the cluster has been deployed, we want to provide the user with a "post install checklist" he can use to finalise configuration prior to schedule workloads

**What should be done**:

- Provision prom / alert manager volumes, using the right example (with block device)
- Change admin password (for k8s API and Grafana)
- Validate cluster deployment (nodes / roles / taints / pods from cli or ui)

**Implementation proposal** (strongly recommended):

**Test plan**:

### PR DESCRIPTION
**Pull request by gdemonet**
**Component**: docs

**Context**: 2.4.2 release

**Summary**:

- change Prom/AM storage example to use `rawBlockDevice` instead of `sparseLoopDevice`
- add section linking to Ops guide for changing credentials
- add some sanity checks with `kubectl` / `etcdctl` (nodes list, pods list, etcd cluster health)

**Acceptance criteria**:

- content is valid and understandable
- no obvious topic is missing from this procedure (sanity checks in particular)

---

Closes: #2117

### CODE REVIEW

File: docs/glossary.rst [hunk 1]
```diff
@@ -58,7 +58,14 @@ Glossary
      The kubelet is the primary "node agent" that runs on each cluster node.
 
      |see K8s docs|
-     `https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/`
+     `kubelet <https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/>`_.
+
+   Namespace
+     A Namespace is a Kubernetes abstraction to support virtual clusters,
```
Comment (TeddyAndrieux) [inline docs/glossary.rst]: `virtual clusters` is a bit "hard" IMO
You should remove this "virtual cluster" part or do like in the kubernetes docs to specify that it's "multiple virtual clusters backed by the same physical cluster"

File: docs/installation/post-install.rst [hunk 2]
```diff
@@ -6,26 +6,173 @@ Post-Installation Procedure
...
-   - Procedure
+   - Sanity check
    - Troubleshooting if needed
+
+
+Changing credentials
+^^^^^^^^^^^^^^^^^^^^
+After a fresh installation, an administration account is created with default
+credentials. It is highly recommended to change those credentials on production
+deployments.
```
Comment (Ebaneck) [inline docs/installation/post-install.rst]: I think you meant to say an `administrator account`?? no?

Maybe make the second sentence a strict recommendation.
`For production deployments, it is highly recommended to change these default credentials`

File: docs/installation/post-install.rst [hunk 3]
```diff
@@ -6,26 +6,173 @@ Post-Installation Procedure
...
+
+To change user credentials and groups for :term:`K8s API <API Server>` (and as
+such, for :ref:`MetalK8s GUI <installation-services-admin-ui>` and
+:term:`SaltAPI`), follow :ref:`this procedure <ops-k8s-admin>`.
+
+To change Grafana user credentials (there is currently no link between K8s API
+users and Grafana users), follow :ref:`this procedure <ops-grafana-admin>`.
```
Comment (Ebaneck) [inline docs/installation/post-install.rst]: I am reluctant about what is in the brackets.. Can we keep it simple. ie

`To change Grafana user credentials, follow ...`

File: docs/installation/post-install.rst [hunk 4]
```diff
@@ -6,26 +6,173 @@ Post-Installation Procedure
...
+   Use the ``kubectl describe node <node_name>`` to get more details about a
+   Node (for instance, to check the right :ref:`taints <node-taints>` are
+   applied).
+
+#. Check that :term:`Pods <Pod>` are in their expected state (most of the time,
+   **Running**, except for Prometheus and AlertManager if the required storage
+   was not provisioned yet - see :ref:`the procedure above <Provision
+   Prometheus Storage>`).
+
+   To look for all Pods at once, use the
```
Comment (Ebaneck) [inline docs/installation/post-install.rst:114]: `To get an overview of all Pods`??

**Review by Ebaneck — CHANGES_REQUESTED** [review]

File: docs/glossary.rst (hunk 1 above)
Comment (gdemonet) [reply docs/glossary.rst]: OK I'll copy paste the "multiple virtual clusters backed by the same physical cluster" from K8s docs ;)

File: docs/installation/post-install.rst (hunk 2 above)
Comment (gdemonet) [reply docs/installation/post-install.rst]: > I think you meant to say an `administrator account`?? no?

Tomato vs. tomato, but ok

> Maybe make the second sentence a strict recommendation.
> `For production deployments, it is highly recommended to change these default credentials`

Stricter would be: `For production deployments, make sure to change these default credentials.`

File: docs/installation/post-install.rst (hunk 3 above)
Comment (gdemonet) [reply docs/installation/post-install.rst]: OK will do

File: docs/installation/post-install.rst (hunk 4 above)
Comment (gdemonet) [reply docs/installation/post-install.rst:114]: Discussed F2F, it's fine. :)

**Review by Ebaneck — APPROVED** [review]

**Pull request merged**